pkg/runtime/runtime.go

```diff
@@ -286,6 +322,20 @@ func (r *RuntimeSpec) fullURL(url string) string {
 	return buildFullURL(url, r.Version, r.devMode)
 }
 
+func shouludInstallFeature(installFeatures []InstallFeature, f InstallFeature) bool {
+	if f == "" {
```

Comment:
maybe add some comment here. What is the meaning of empty `InstallFeature`? (I know but the next reader might find it hard to understand).
also maybe instead of `installFeatures` call it `featuresToInstall` and instead of `f` call it `featureName`